packages/workers/project.json

```diff
@@ -55,7 +55,10 @@
       "executor": "nx:run-commands",
       "options": {
         "color": true,
-        "command": "docker-compose run --rm --no-deps --entrypoint /bin/bash workers /app/packages/workers/scripts/runtime/run_lint.sh"
+        "commands": [
+          "docker-compose run --rm --no-deps --entrypoint /bin/bash workers /app/packages/workers/scripts/runtime/run_lint.sh",
+          "pnpm nx run workers:lint:js"
```

Comment:
@mkleszcz Let me know what do you think about this change, please. For some reason eslint config couldn't resolve the path to root config that is extended when run from the bash script.